utils/run_test.sh

```diff
@@ -79,6 +79,31 @@ if [ -d "/mnt/daos" ]; then
         SL_PREFIX=$PWD/${SL_PREFIX/*\/install/install}
     fi
 
+    echo "Running Cmocka tests"
+    if [ -z "$RUN_TEST_VALGRIND" ]; then
+        # Tests that do not run valgrind
+        run_test src/rdb/raft_tests/raft_tests.py
+        go_spdk_ctests="${SL_PREFIX}/bin/nvme_control_ctests"
+        if test -f "$go_spdk_ctests"; then
+            run_test "$go_spdk_ctests"
+        else
+            echo "$go_spdk_ctests missing, SPDK_SRC not available when built?"
+        fi
+        run_test src/control/run_go_tests.sh
+    else
+        if [ "$RUN_TEST_VALGRIND" = "memcheck" ]; then
+            [ -z "$VALGRIND_SUPP" ] &&
+                VALGRIND_SUPP="$(pwd)/utils/valgrind_memcheck.supp"
+            VALGRIND_CMD="valgrind --leak-check=full --show-reachable=yes \
```

Comment:
we need to generate xml output --xml=yes --xml-file=[unique file name]

Reply:
@jolivier23  by unique file name, you mean one xml file per run_test? or would it be okay one xml for all of them?

Reply:
I don't know if you can do one for all of them.  I think it just gets overwritten in such case.   If it appends, then it depends on whether Jenkins can interpret it properly

Reply:
it will overwrite the file.  If you're not able to generate unique filenames yourself use --xml=yes --xml=file=run_test-%p.memcheck.xml and it'll embed the pid in the filename.  You can they use *memcheck.xml as a wildcard in the Jenkinsfile.